yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -120,6 +120,7 @@ func (conv *DebeziumValueConverter) ConvertEvent(ev *tgtdb.Event, table string)
 func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*string) error {
 	for column, value := range m {
 		if value == nil {
+			m[column] = new(string)
```

Comment:
I'd suggest declare globally NULL_STRING = "NULL" and assign the address to the pointer. 
That way, we can avoid creating a new string for every row.

Reply:
good point. Will do that change